turbopack/crates/turbopack-nft/src/nft.rs

```diff
@@ -102,7 +102,7 @@ async fn node_file_trace_operation(
         .cell(),
         ResolveOptionsContext {
             enable_node_native_modules: true,
-            enable_node_modules: Some(input_dir.clone()),
+            enable_node_modules: Some(ReadRef::into_owned(input_dir.clone())),
```

Comment:
Why `ReadRef::into_owned` as opposed to cloning the RcStr?

Reply:
hmm, for some reason i thought the reference here was conditional so the clone earlier wasn't always needed, but that isn't true.

There is an unnecessary clone here, but i removed the wrong one